src/drivers/intel/haswell/ssp.c

```diff
@@ -347,7 +349,7 @@ static inline int ssp_set_config(struct dai *dai,
 	else
 		sscr0 |= SSCR0_DSIZE(data_size);
 
-	sscr1 |= SSCR1_TFT(0x7) | SSCR1_RFT(0x7);
+	sscr1 |= SSCR1_TFT(0x8) | SSCR1_RFT(0x8);
```

Comment:
we should try to make this smarter and depend on the number of slots (for both BYT and BDW).